integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -230,14 +247,20 @@ shortCircuit:
   # local means the cache directory is in the same mount namespace,
   # uuid means interact with domain socket
   policy: uuid
-  # Attributes if the domain socket volume is PVC
+  # volumeType controls the type of shortCircuit volume.
+  # It can be "persistentVolumeClaim" or "hostPath"
+  volumeType: persistentVolumeClaim
+  size: 1Mi
+  # Attributes to use if the domain socket volume is PVC
   pvcName: alluxio-worker-domain-socket
-  pvcAccessModes:
+  accessModes:
```

Comment:
Please note this and any other renames in changelog. A rename will break back compatibility, avoid if not absolutely necessary

Reply:
Actually i found this `accessModes` i'm changing to is the one used in the docs. I realized this is changed to `pvcAccessModes` sometime in 2.3 snapshot. So I think it's okay.

Reply:
yep